scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1169,37 +1166,37 @@ private void build() {
 
     private Boolean isExecutionCalendarGIDefined(String jobDescriptor) {
         Document dom = XMLParser.parse(jobDescriptor);
-        dom.getDocumentElement().normalize();
-
         Boolean exists = false;
```

Comment:
Just forgot to say that you no longer use this flag (exists). you should consider removing it